workflow/resources/datavzrd/samples-template.yaml

```
@@ -0,0 +1,15 @@
+__use_yte__: true
+
+name: ?f"Inputs used to start the pipeline"
```

This could be a bit more specific to the `samples.tsv`, maybe:
```suggestion
name: ?f"Sample annotations used in modelling differential expression and in plots"
```